web/src/components/tasks/business-formation/BusinessFormationPaginator.test.tsx

```diff
@@ -186,6 +186,14 @@ describe("<BusinessFormationPaginator />", () => {
       });
     });
 
+    it("persists the error state on step one even after switching steps", async () => {
+      const page = preparePage(initialUserData, displayContent);
+      page.fillText("Search business name", "Pizza Joint");
+      await page.searchBusinessName({ status: "UNAVAILABLE" });
+      await page.stepperClickToBusinessStep();
+      expect(page.getStepStateInStepper(LookupStepIndexByName("Name"))).toEqual("ERROR");
+    });
```

Comment:
This test is labelled "even after" switching steps but it is ONLY testing what happens after switching steps - not what happens before switching steps.
- if it's valuable to test BOTH before AND after switching steps, this needs another assertion before the switch
- if its only testing AFTER then then "even" should be dropped from the label
(i suspect it's the first one)